test/parallel/test-tls-canonical-ip.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+require('../common');
+
+// Test conversion of IP addresses to the format returned
+// for addresses in Subject Alternative Name section
+// of a TLS certificate
+
+const assert = require('assert');
+const canonicalizeIP = process.binding('cares_wrap').canonicalizeIP;
```

Comment:
Nit: it would be nice to use destructuring here since there was a PR recently to convert most of the code base to more consistently do that.